### PR TITLE
Add new ApiRest documentation

### PR DESCRIPTION
**Pull request by devsigner**

### CODE REVIEW

File: _pages/api/rest/v1.html [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+</div><div class="middle"><section id="authentication" class="resource-group"><h2 class="group-heading">Authentication <a href="#authentication" class="permalink">¶</a></h2><p>The Augment RESTFULL API use the Oauth process for authentication.
```
Comment (sgara) [inline _pages/api/rest/v1.html]: Any reason to use HTML over Markdown?

File: _layouts/apib.html [hunk 2]
```diff
@@ -0,0 +1,1316 @@
...
+    // Remove the `preload` class to enable animations
+    document.querySelector('body').className = '';
+};
+
+    $('.apib-container .container-fluid.triple nav').affix({
+      offset: {
+        top: 370
+      }
+    })
+</script>
```
Comment (sgara) [inline _layouts/apib.html]: I'd rather split this file into scss + include files and keep only layout architecture here

File: _pages/api/rest/v1.html (hunk 1 above)
Comment (devsigner) [reply _pages/api/rest/v1.html]: Yes is generated from ApiBlueprint format to html. That's why the html is ugly ;)

File: _layouts/apib.html (hunk 2 above)
Comment (devsigner) [reply _layouts/apib.html]: Yes the css is now extract in scss file

**Pull request merged**